example/frontend-testing/websocket.js

```diff
@@ -0,0 +1,80 @@
+// WebSocket URL
+const WS_URL = "ws://localhost:8394";
+let audioContext, audioProcessorNode, socket;
+let isStreaming = false;
+
+document.getElementById("startRecording").addEventListener("click", startRecording);
+document.getElementById("stopRecording").addEventListener("click", stopRecording);
+
+async function startRecording() {
+  // Disable the Start button and enable the Stop button
+  document.getElementById("startRecording").disabled = true;
+  document.getElementById("stopRecording").disabled = false;
+
+  // Initialize WebSocket
+  socket = new WebSocket(WS_URL);
+
+  socket.onopen = async () => {
+    console.log("WebSocket connection established");
+
+    // Initialize AudioContext and Audio Worklet
+    audioContext = new AudioContext({ sampleRate: 16000 });
```

Comment:
do you also have settings for the amount of channels or sample size here? If not, does not matter as long as your processor gets it right :)

Reply:
removed redundant option